ted2zim/templates/assets/db.js

```diff
@@ -4,109 +4,109 @@
  */
 
  /* exported videoDB */
-var videoDB;
```

Comment:
why the reformating of this file? Not that it's a bad thing but please explain it. Also, if you're using a code formatter, mention it in a comment on top of file so other maintainers can cope with it.

Reply:
Yup. You're right. I'm reverting to the original content for now (seems like it happened due to a huge copy-paste). Maybe what we should do is to have all JS files formatted in another PR. Maybe it can be just for code formatting, adding docstrings, renaming variables and basically anything that improves code readability and quality. Doing this in this PR may lead it to be difficult to review as formatting would change many lines.

Reply:
👍